torchchat/model.py

```diff
@@ -34,19 +38,147 @@
 
 from torchchat.utils.build_utils import find_multiple, get_precision
 
+from torchtune.models.flamingo import flamingo_decoder, flamingo_vision_encoder
+from torchtune.models.llama3_1._component_builders import llama3_1 as llama3_1_builder
+from torchtune.modules.model_fusion import DeepFusionModel
+from torchtune.models.clip import clip_vision_encoder
+
 config_path = Path(f"{str(Path(__file__).parent)}/model_params")
 
 
+class QuickGELUActivation(nn.Module):
+    """
+    Applies GELU approximation that is fast but somewhat inaccurate. See: https://github.com/hendrycks/GELUs
+    """
+
+    def forward(self, input):
+        return input * torch.sigmoid(1.702 * input)
+
+
 def identity(**kwargs):
     if len(kwargs) != 1:
         raise ValueError("Only one argument is expected")
     return list(kwargs.values())[0]
 
 
+@dataclass
+class ProjectorArgs:
```

Comment:
Let's add docstring for each of these dataclasses since they not the usuall Llama classes

Reply:
I think I can remove it rn. We can further take it back when we design arg class for different modules.